app/assets/stylesheets/teams.css

```diff
@@ -3,12 +3,173 @@
   They will automatically be included in application.css.
 */
 
-label.required:after {
-  content: "*";
-  vertical-align: 0.5em;
-  font-size: 0.75em;
-  font-weight: normal;
```

Comment:
the css you removed here affected the `edit` team page, adding an asterisk after required fields, as specified in #218.
please add again!
<img width="1066" alt="screen shot 2018-02-02 at 08 58 24" src="https://user-images.githubusercontent.com/9486619/35722856-490b2ed4-07f8-11e8-8cbc-9e0ee51236a2.png">

Reply:
Will fix

Reply:
Done.